pyproject.toml

```diff
@@ -134,6 +137,8 @@ filterwarnings = [
     "ignore:The unit 'Angstrom' has been deprecated in the VOUnit standard\\. Suggested.* 0\\.1nm\\.",
     "ignore:((.|\n)*)Sentinel is not a public part of the traitlets API((.|\n)*)",
     "ignore:datetime\\.datetime\\.utcfromtimestamp:DeprecationWarning",  # asdf + dateutil<=2.8.2 + Python 3.12
+    "ignore:'audioop' is deprecated and slated for removal in Python 3.13",
+    "ignore:Importing display from IPython.core.display is deprecated since IPython 7.14, please import from IPython.display",
```

Comment:
where does this need to be fixed?  Is there an upstream issue?

Reply:
Same as above.

Reply:
#3332